WS2012R2/lisa/xml/UDP_SRIOV_v4.xml

```diff
@@ -42,83 +42,68 @@
-        <param>CONNECTIONS_PER_IPERF3=128</param>
-        <param>TEST_SIGNAL_FILE=iperf3.test.signal</param>
-        <param>TEST_RUN_LOG_FOLDER=iperf3-test-logs_1k</param>
         <param>SSH_PRIVATE_KEY=rhel5_id_rsa</param>
-        <param>IPERF3_TEST_CONNECTION_POOL=(1 2 4 8 16 32 64 128 256 512 1024)</param>
       </testParams>
       <uploadFiles>
-        <file>iPerf3_Client_Logs.zip</file>
-        <file>iPerf3_Panorama_ServerSideScript.log</file>
-        <file>iperf3-UDP-1k.log</file>
+        <file>logs_ipv4.zip</file>
       </uploadFiles>
       <timeout>14400</timeout>
       <OnError>Continue</OnError>
     </test>
     <test>
       <testName>UDP_v4_8k_sriov</testName>
       <pretest>setupScripts\NET_SendIPtoVM.ps1</pretest>
-      <!--<setupScript>
-                 <file>setupScripts\Check_PowerPlan.ps1</file>
-                <file>setupscripts\perf_SRIOV_enable.ps1</file>
-            </setupScript>-->
-      <testScript>perf_iperf_client.sh</testScript>
-      <files>remote-scripts/ica/perf_iperf_client.sh,remote-scripts/ica/perf_iperf_server.sh,remote-scripts/ica/utils.sh,remote-scripts/ica/perf_utils.sh</files>
-      <files>Tools/iperf-3.1.3.tar.gz</files>
+      <setupScript>
+        <file>setupScripts\perf_SRIOV_enable.ps1</file>
+      </setupScript>
+      <testScript>perf_ntttcp_client.sh</testScript>
+      <files>remote-scripts/ica/perf_ntttcp_client.sh,remote-scripts/ica/perf_ntttcp_server.sh,remote-scripts/ica/utils.sh,remote-scripts/ica/perf_utils.sh</files>
       <testParams>
-        <param>TC_COVERED=IPv4</param>
         <param>SRIOV=yes</param>
+        <param>UDP_MODE=yes</param>
+        <param>TC_COVERED=IPv4</param>
         <param>NETWORK_NAME=SRIOV</param>
         <param>SnapshotName=ICABase</param>
-        <param>IPERF_PACKAGE=iperf-3.1.3.tar.gz</param>
-        <param>STATIC_IP=10.10.10.20</param>
-        <param>IPERF3_SERVER_IP=10.10.10.10</param>
+        <param>STATIC_IP=192.168.12.3</param>
+        <param>SERVER_IP=192.168.12.4</param>
         <param>NETMASK=255.255.255.0</param>
         <param>SERVER_OS_USERNAME=root</param>
-        <param>BANDWIDTH=40g</param>
+        <param>TEST_DURATION=60</param>
+        <param>NTTTCP_BUFFER=8k</param>
+        <param>TEST_THREADS=(1 2 4 8 16 32 64 128 256 512 1024 2048 4096 6144 8192 10240)</param>
+        <param>MAX_THREADS=64</param>
         <param>ETH_NAME=eth1</param>
         <param>MAC=001600444416</param>
-        <param>IPERF3_BUFFER=8k</param>
+        <param>log_folder=logs_ipv4</param>
         <param>VM2NAME=SRIOV-Server</param>
-        <param>VM2SERVER=LIS-PERF09</param>
-        <param>REMOTE_SERVER=LIS-PERF09</param>
-        <param>IPERF3_PROTOCOL=UDP</param>
-        <param>INDIVIDUAL_TEST_DURATION=60</param>
-        <param>CONNECTIONS_PER_IPERF3=128</param>
-        <param>TEST_SIGNAL_FILE=iperf3.test.signal</param>
-        <param>TEST_RUN_LOG_FOLDER=iperf3-test-logs_8k</param>
+        <param>VM2SERVER=43f13-36</param>
```

Comment:
please default to 43f13-36 in all places in the xml, all references for perf09 should be removed from this xml

Reply:
Updated